snap/snapcraft.yaml

```diff
@@ -153,7 +153,8 @@ parts:
 
   freecad:
     plugin: cmake
-    source: https://github.com/FreeCAD/FreeCAD.git
+    source: https://github.com/ein-shved/FreeCAD.git
+    source-branch: about-dev-painted
```

Comment:
This branch does exist, see: https://github.com/ein-shved/FreeCAD/tree/about-dev-painted